apollo-runtime/src/test/java/com/apollographql/apollo/ApolloExceptionTest.java

```diff
@@ -31,7 +31,7 @@
 import static com.google.common.truth.Truth.assertThat;
 
 @SuppressWarnings("unchecked") public class ApolloExceptionTest {
-  private static long TIMEOUT_SECONDS = 2;
+  private static long timeoutSeconds = 2;
```

Comment:
is there a reason this file updated?

Reply:
had to make unrelated changes to make lint checks pass. sounds like they have been updated in CI though so i will revert all the unrelated changes